README.md

```diff
@@ -81,6 +91,41 @@ $ rpc-client/build/install/rpc-client/bin/rpc-client issue -u localhost:10011 -n
 
 The client will discover all the participants in the network and prompt you for a node as the counterprise (borrower), make sure to pick the other node you created in the network, not the node you are connected to or the notary.
 
+### Account Utilities
+
+#### Create new account
+To create a new account with name `partyA` on a node:
+```
+$ rpc-client/build/install/rpc-client/bin/rpc-client account-utils --create-account --account-name partyA -u localhost:10011 -n user1 -p test
+```
+#### Generate a key pair for a given account
+To create a new key pair for a given account on a node:
+```
+$ rpc-client/build/install/rpc-client/bin/rpc-client account-utils --create-account-key --account-uuid 85b8121c-2bfb-4bec-af36-41a33d87b897 -u localhost:10011 -n user1 -p test
+```
+#### Share account's information with other node
+To share account info with other node so, that they can use it in a flow:
+```
+$ rpc-client/build/install/rpc-client/bin/rpc-client account-utils --share-account-info --account-name partyA  --party-share-to zzgqil3zm1 -u localhost:10011 -n user1 -p test
+```
+
+### Test Issuance of an IOU
+To create a new IoU from the lender and have the borrower sign it and for the notary to notarize it using `account <> account`:
+```
+$ rpc-client/build/install/rpc-client/bin/rpc-client issue --lender-account partyA --borrower-account partyB --value 98 -u localhost:10011 -n user1 -p test
+```
+or To using `node <> node`
+```
+$ rpc-client/build/install/rpc-client/bin/rpc-client issue  --value 98 -u localhost:10011 -n user1 -p test
+```
+The client will discover all the participants in the network and prompt you for a node as the lender and  a counterparty (borrower), make sure to pick the other node you created in the network, not the node you are connected to or the notary.
+
+or To using `node <> account`
+```
+$ rpc-client/build/install/rpc-client/bin/rpc-client issue  --borrower-account partyB --value 98 -u localhost:10011 -n user1 -p test
+```
+The client will discover all the participants in the network and prompt you for a node as the lender and  a counterparty (borrower), make sure to pick the other node you created in the network, not the node you are connected to or the notary.
```

Comment:
the borrower prompt should be omitted here as the borrower account has already been specified right?

Reply:
Account name is not unique network wide. two accounts with different names can exist in different nodes. So distinguish which account to use, prompt is used.